applications/CompressiblePotentialFlowApplication/python_scripts/potential_flow_solver.py

```diff
@@ -169,6 +169,7 @@ def SetEchoLevel(self, level):
     def Clear(self):
         self.incompressible_solution_stratety.Clear()
 
+    @staticmethod
```

Comment:
does this still work?

I am not sure, I would no put it

Don't believe everything codacy suggests, it is not always applicable ;)

Reply:
Oh Ok, I thought that to merge all of the checks must be successful. Anyways it still works, but I will put it back anyways.